lib/search.js

```diff
@@ -9,13 +10,17 @@ const octokit = require('@octokit/rest')({
   },
 })
 
-// GitHub search paramaters for the Node.js org
+// GitHub search parameters for the Node.js org
```

Comment:
Heh, `for the Node.js org` is an artifact of the original implementation. Want to strike it?

Reply:
my spell checker was throwing a warning
so as google :stuck_out_tongue:

Reply:
Yeah I apparently don't have a speel checker in VS Code 😅

Reply:
also grammarly for chrome :stuck_out_tongue: